src/timestepper.cpp

```diff
@@ -572,7 +582,8 @@ void ExplEuler::evolveBWD(const double tstop,const  double tstart,const  Vec x,
 
 }
 
-ImplMidpoint::ImplMidpoint(MapParam config,MasterEq* mastereq_, int ntime_, double dt_, LinearSolverType linsolve_type_, int linsolve_maxiter_, Output* output_, bool storeFWD_, MPI_Comm comm_time_) : TimeStepper(config, mastereq_, ntime_, dt_, output_, storeFWD_, comm_time_) {
+ImplMidpoint::ImplMidpoint(MapParam config,MasterEq* mastereq_, int ntime_global_, int ntime_, double dt_, LinearSolverType linsolve_type_, int linsolve_maxiter_, Output* output_, bool storeFWD_, MPI_Comm comm_time_)
+  : TimeStepper(config, mastereq_, ntime_global_, ntime_, dt_, output_, storeFWD_, comm_time_) {
```

Comment:
Only white space added

Reply:
If this indentation is not your preference, we can change it.

Reply:
See my previous reply

Reply:
Revoked the line change